### PR TITLE
Adding data to project public page + a few tweaks

### PR DESCRIPTION
**Pull request by SARodrigues**
This PR finishes the connection to the endpoint for the data displayed in the project public page.

**Notes:**  
- We're missing the total impact score (not yet available)
  _Currently a placeholder is displayed_  
- Adding the geometry in the map is tracked by [LET-592](https://vizzuality.atlassian.net/browse/LET-592)
- Fixed an issue where the project developer card would error if no project developer is provided  
  - This was caused by a separate issue in which the API may not return `involved_project_developers`   
- Contacts section to display all project developers' contacts and not only the main one  
  _Assuming the user has permission to see them and said contacts are available_  
- Minor styling/responsiveness tweaks  
- Priority landscape not added yet
  _not part of the user story yet_

## Testing instructions

Verify that there is no hardcoded information. 

## Tracking

[LET-591](https://vizzuality.atlassian.net/browse/LET-591)
[LET-626](https://vizzuality.atlassian.net/browse/LET-626)

### CODE REVIEW

**Comment by SARodrigues** [comment]
Rebased on `develop` to fix conflicts.

File: frontend/containers/project-page/funding/component.tsx [hunk 1]
```diff
@@ -70,13 +84,13 @@ export const Funding: React.FC<FundingProps> = ({ project }: FundingProps) => {
             </h3>
             <div className="flex flex-col font-sans lg:space-x-20 lg:flex-row">
               <div className="flex flex-col space-y-1">
-                <p className="text-xl font-semibold lg:text-2xl">$ 24000</p>
+                <p className="text-xl font-semibold lg:text-2xl">{ticketSizeStr}</p>
```
Comment (SARodrigues) [inline frontend/containers/project-page/funding/component.tsx:87]: Note to self: this is incorrect. This is not the ticket size. 🤦

File: frontend/containers/project-page/contact/component.tsx [hunk 2]
```diff
@@ -1,59 +1,64 @@
...
-  const contact: ContactItemType = {
-    name: project?.project_developer.name,
-    email: email,
-    phone: phone,
-  };
-  // TODO: User should also have permissions to see contact info
+
+  const contacts = useMemo(
+    () =>
+      [project?.project_developer, ...project?.involved_project_developers]
```
[the remark this answers is not in the document]
Comment (barbara-chaves) [reply frontend/containers/project-page/contact/component.tsx]: I change it on a commit

**Review by barbara-chaves — CHANGES_REQUESTED** [review]

**Review by barbara-chaves — APPROVED** [review]
Perfect! Just missing the pd image on the header's contact info modal.

**Pull request merged**